content/posts/o-que-eu-aprendi-depois-de-dar-100-palestras.md

```diff
@@ -0,0 +1,87 @@
+---
+title: O que eu aprendi depois de dar 100 palestras
+author: Zeno Rocha
+type: post
+date: 2017-11-01
+excerpt: Olá! Meu nome é Zeno Rocha. Ao longo dos últimos anos, aprendi muito sobre dar palestras e agora eu to aqui para compartilhar essas lições.
```

Comment:
eu trocaria por 'estou', ou se for ser mais informal mesmo, coloque o acento circunflexo: 'tô'